modules/popup-stack/lib/PopupStack.ts

```diff
@@ -183,13 +186,23 @@ const setToWindow = (path: string, value: any) => {
 // defined on the page, we need to use that one. Never, ever, ever change this variable name on
 // window
 const stack: Stack = getFromWindow('workday.__popupStack') || {
-  description: 'Global popup stack from @workday/canvas-kit-popup-stack',
-  items: [] as PopupStackItem[],
+  description: 'Global popup stack from @workday/canvas-kit/popup-stack',
+  container: () => document.body,
+  items: [],
   zIndex: {min: 30, max: 50, getValue: getValue},
-  _adapter: {} as Partial<typeof PopupStack>,
+  _adapter: {},
 };
 setToWindow('workday.__popupStack', stack);
```

Comment:
That's possible, but a single stack needs to be created as the base stack and that needs to be shared. It could be that the first import of `PopupStack` is a version before this change and this variable would still exist. It could be that this code is removed and an old version that uses this still adds the global variable. It is important that all instances of this file use the same reference.

It is possible to deprecate, but that would take a lot of thought and testing. I have no plans to remove it at the moment.